Model/Sync/Reset/Orders.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync\Reset;
+
+class Orders extends Main
+{
+    const ORDERS_SYNC_TABLE = 'yotpo_orders_sync';
+    const CRONJOB_CODES = ['yotpo_cron_core_orders_sync'];
+
+    const ORDERS_TABLE = 'sales_order';
+    const ORDERS_DATA_LIMIT = 300;
+    const YOTPO_ENTITY_NAME = 'order';
+    /**
+     * @return array <string>
+     */
+    protected function getTableResourceNames()
+    {
+        return [self::ORDERS_SYNC_TABLE];
+    }
+
+    /**
+     * @return array <string>
+     */
+    protected function getCronJobCodes()
+    {
+        return self::CRONJOB_CODES;
+    }
+
+    /**
+     * @return string
+     */
+    public function getYotpoEntityName()
+    {
+        return self::YOTPO_ENTITY_NAME;
+    }
+
+    /**
+     * @param int $storeId
+     * @param boolean $skipSyncTables
+     * @return void
+     * @throws \Zend_Db_Statement_Exception
+     */
+    public function resetSync($storeId, $skipSyncTables = false)
+    {
+        parent::resetSync($storeId, true);
+        $this->clearSyncTracks($storeId);
+        $this->setResetInProgressConfig($storeId, '0');
+    }
+
+    /**
+     * @param int $storeId
+     * @return void
+     */
+    private function clearSyncTracks($storeId)
+    {
+        $connection  = $this->resourceConnection->getConnection();
+        $table = $this->resourceConnection->getTableName(self::ORDERS_TABLE);
+        $select = $connection->select()
+            ->from($table, 'entity_id')
+            ->where('store_id', $storeId);
+
+        $offset = 0;
+        do {
+            $entityIds = $connection->fetchCol($select->limit(self::ORDERS_DATA_LIMIT, $offset));
```

Comment:
Why did you choose here an approach that is different than what we have with, for example, the catalog sync?

Reply:
We will change this as we do for catalog sync and update the PR